.github/workflows/rust.yml

```diff
@@ -0,0 +1,44 @@
+name: Build and Deploy Site
+
+env:
+  NODE_VERSION: "12.x"
```

Comment:
Let's use 14

Reply:
Changed to latest LTS. 

![dangerous](https://media.giphy.com/media/DwIdasRkFKsMg/giphy.gif)